src/IB/IBFEMethod.cpp

```diff
@@ -1054,7 +1138,14 @@ IBFEMethod::computeLagrangianForce(const double data_time)
         {
             computeStressNormalization(*d_Phi_half_vecs[part], *d_X_half_vecs[part], data_time, part);
         }
+        // Zero-out force vector
+        d_F_half_vecs[part]->zero();
         computeInteriorForceDensity(*d_F_half_vecs[part], *d_X_half_vecs[part], d_Phi_half_vecs[part], data_time, part);
+        if (d_direct_forcing_kinematics_data[part])
+        {
+            d_direct_forcing_kinematics_data[part]->computeLagrangianForce(
```

Comment:
Should these forces should go into a zero_clone() of the force vector, and then get added to the force vector owned by this class. That way implementations of the kinematics class don't have to remember to accumulate.

Reply:
That also sounds good. I can change it. Let me know.

Reply:
Yes please do make this change.